include/openssl/ssl.h

```diff
@@ -1910,18 +1919,45 @@ int DTLSv1_listen(SSL *s, BIO_ADDR *client);
  * NOTE: A side-effect of setting a CT callback is that an OCSP stapled response
  *       will be requested.
  */
-__owur int SSL_set_ct_validation_callback(SSL *s,
-                                          ct_validation_cb callback,
-                                          void *arg);
-__owur int SSL_CTX_set_ct_validation_callback(SSL_CTX *ctx,
-                                              ct_validation_cb callback,
-                                              void *arg);
+int SSL_set_ct_validation_callback(SSL *s, ssl_ct_validation_cb callback,
+                                   void *arg);
+int SSL_CTX_set_ct_validation_callback(SSL_CTX *ctx,
+                                       ssl_ct_validation_cb callback,
+                                       void *arg);
+#define SSL_disable_ct(s) \
+        ((void) SSL_set_validation_callback((s), NULL, NULL))
+#define SSL_CTX_disable_ct(ctx) \
+        ((void) SSL_CTX_set_validation_callback((ctx), NULL, NULL))
+
+/*
+ * The ssl_ct_validation_t type enumerates the available behaviours of the
+ * built-in SSL CT validation callback selected via SSL_enable_ct() and
+ * SSL_CTX_enable_ct().  The underlying callback is a static function in
+ * libssl.  The requested validation type is also an index into an array of
+ * callback args used by the callback to select the appropriate policy, so
+ * these need to be consecutive non-negative integers starting at zero.
+ */
+enum {
+    SSL_CT_VALIDATION_PERMISSIVE = 0,
+    SSL_CT_VALIDATION_STRICT
+};
```

Comment:
It'd be nicer if this enum was named and `SSL_enable_ct` and `SSL_CTX_enable_ct` used it, rather than `int`, as a parameter.